src/app.css

```diff
@@ -1,29 +1,4 @@
-.wrapper {
-  height: 100vh;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-}
-
-h1 {
-  margin-bottom: 0;
-}
-
-p, .counter {
-  padding-bottom: 2rem;
-}
-
-.counter {
-  display: flex;
-  flex-direction: row;
-  align-items: center;
-}
-
-.counter-value {
-  margin: 0 8px;
-}
-
-.reset {
-  margin-left: 15px;
-}
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300&family=Raleway:wght@400;700&display=swap');
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
```

Comment:
what do these lines of code in app.css do?

Reply:
We are importing the tail wind css utilities to be included in our project through our css